redaxo/src/addons/install/lang/de_de.lang

```diff
@@ -64,11 +64,14 @@ install_settings_general = Allgemein
 install_settings_backups = Backups erstellen
 install_settings_backups_note = Vor dem Update von Paketen durch den Installer wird jeweils eine Kopie der aktuellen Version in <code>redaxo/data/addons/install/</code> abgelegt. Diese können bei Fehlern wieder eingespielt werden.<br>Bitte beachten, dass die Datenbank dabei nicht gesichert wird!
 install_settings_myredaxo_account = myREDAXO-Account
+install_settings_myredaxo_notice = Der API-Key wird benötigt, um den Upload deiner AddOns zur REDAXO-Website durchzuführen. Im Installer-AddOn kannst Du auch verschiedene Versionen deiner AddOns erstellen sowie zusätzlich auch Offline-AddOns einsehen.
```

Comment:
Ich würde in dem Kontext nicht von offline sprechen, da sie ja online sind. Ich würde den Begriff "versteckt" oder "nur für den Account sichtbar" o.ä. wählen.

Reply:
Die AddOns sind offline und ohne API-Key nicht im Installer zu einzusehen.

Reply:
Offline verstehe ich als lokal, ohne Internet erreichbar oder eben nicht (online, im Internet) verfügbar.

Reply:
Da die Checkbox nun mal "online" heißt, denke ich, passt es schon, dann hier von "offline" in dem Erklärtext zu sprechen.
Ob man das Feld an sich umbenennt, wäre eine andere Diskussion.